rust/rsmgp-module-template/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+use rsmgp_sys::memgraph::*;
+use rsmgp_sys::mgp::*;
+use rsmgp_sys::result::*;
+use rsmgp_sys::rsmgp::*;
+use rsmgp_sys::{close_module, define_procedure, init_module};
+use std::ffi::CString;
+use std::os::raw::c_int;
+use std::panic;
```

Comment:
TODO(@gitbuda): Somehow simplify use statements and the beginning of each Rust query module.